stellar/deploy-contract.js

```diff
@@ -50,6 +50,14 @@ async function getInitializeArgs(config, chain, contractName, wallet, options) {
         case 'interchain_token_service': {
             const gatewayAddress = nativeToScVal(Address.fromString(chain?.contracts?.axelar_gateway?.address), { type: 'address' });
             const gasServiceAddress = nativeToScVal(Address.fromString(chain?.contracts?.axelar_gas_service?.address), { type: 'address' });
+            const hubChain = config.chains.axelar || config.axelar;
+            const itsHub = hubChain.contracts.InterchainTokenService.address;
```

Comment:
You could use `validateParameters` for validation check. Reference: https://github.com/axelarnetwork/axelar-contract-deployments/blob/04a0dbd71cf2cda3e4791e8363f364379ae53830/stellar/deploy-contract.js#L148